src/components/Tree/Tree.tsx

```diff
@@ -195,6 +195,7 @@ export const Tree = memo(
         draggable = false,
         multiselect = false,
         dragHandlerPosition = 'left',
+        disableDragDelay = false,
```

Comment:
```suggestion
        dragDelay = 150,
```
Try to avoid negation in props